src/Misc/PresetExtractor.cpp

```diff
@@ -227,7 +227,10 @@ void doPaste(MiddleWare &mw, string url, string type, XMLwrapper &xml, Ts&&... a
         type = "Plfo";
 
     if(xml.enterbranch(type) == 0)
+    {
+        delete t;
```

Comment:
Same comment applies to the old code, but should we consider this a bug if we ever get to this return statement?

Reply:
I think so, but what is our way to handle bugs? Should zyn exit just because of a failed paste?

Another thing we could do is print a message on `return`ing, so that the problem is revealed.